conformant = "Partial"
notes = """
Incorrectly reports error for type alias defined with ParamSpec.
Does not report invalid specialization of generic type alias with bound TypeVar.
Incorrectly evaluates generic type alias with ParamSpec with missing type argument.
Does not report some illegal annotation forms as invalid type aliases.
Does not report invalid specialization of already-specialized generic type alias.
"""
output = """
File "aliases_explicit.py", line 41, in <module>: Invalid type annotation 'Callable[Concatenate[int, P], R][[str, str], None]'  [invalid-annotation]
  Callable[Concatenate[int, P], R] expected 1 parameter, got 2
File "aliases_explicit.py", line 41, in <module>: Invalid type annotation '[str, str]'  [invalid-annotation]
  Not a type
File "aliases_explicit.py", line 57, in good_type_aliases: Callable[Concatenate, Any] [assert-type]
  Expected: Callable[[int, str, str], None]
    Actual: Callable[Concatenate, Any]
File "aliases_explicit.py", line 60, in good_type_aliases: Callable[[Any], None] [assert-type]
  Expected: Callable[..., None]
    Actual: Callable[[Any], None]
File "aliases_explicit.py", line 67, in <module>: Invalid type annotation 'Optional[int][int]'  [invalid-annotation]
  Optional[int] expected 0 parameters, got 1
File "aliases_explicit.py", line 68, in <module>: Invalid type annotation 'List[Optional[int]][int]'  [invalid-annotation]
  List[Optional[int]] expected 0 parameters, got 1
File "aliases_explicit.py", line 69, in <module>: Invalid type annotation 'List[T][int, int]'  [invalid-annotation]
  List[T] expected 1 parameter, got 2
File "aliases_explicit.py", line 70, in <module>: Invalid type annotation 'Callable[[int, T], T][int, int]'  [invalid-annotation]
  Callable[[int, T], T] expected 1 parameter, got 2
File "aliases_explicit.py", line 71, in <module>: Invalid type annotation 'Callable[Concatenate[int, P], R][int, int]'  [invalid-annotation]
  Callable[Concatenate[int, P], R] expected 1 parameter, got 2
File "aliases_explicit.py", line 80, in <module>: Invalid type annotation '[int, str]' for BadTypeAlias2 [invalid-annotation]
  Not a type
File "aliases_explicit.py", line 81, in <module>: Invalid type annotation '((int, str),)' for BadTypeAlias3 [invalid-annotation]
  Not a type
File "aliases_explicit.py", line 82, in <module>: Invalid type annotation '<instance of List[Type[int]]>' for BadTypeAlias4 [invalid-annotation]
  Not a type
File "aliases_explicit.py", line 83, in <module>: Invalid type annotation "{'a': 'b'}" for BadTypeAlias5 [invalid-annotation]
  Not a type
File "aliases_explicit.py", line 87, in <module>: Invalid type annotation '3' for BadTypeAlias9 [invalid-annotation]
  Not a type
File "aliases_explicit.py", line 88, in <module>: Invalid type annotation 'True' for BadTypeAlias10 [invalid-annotation]
  Not a type
File "aliases_explicit.py", line 89, in <module>: Invalid type annotation '1' for BadTypeAlias11 [invalid-annotation]
  Not a type
File "aliases_explicit.py", line 91, in <module>: Invalid type annotation '<instance of str>' for BadTypeAlias13 [invalid-annotation]
  Must be constant
File "aliases_explicit.py", line 102, in <module>: Invalid type annotation 'Union[list, set][int]'  [invalid-annotation]
  Union[list, set] expected 0 parameters, got 1
"""
